src/webpack.js

```diff
@@ -29,13 +32,138 @@ const parseEntrypoint = (entrypoint) => {
   };
 };
 
+const cwd = process.cwd();
+
+/**
+ * Helper function to trim absolute file path by removing the `process.cwd()`
+ * prefix if file is nested under `process.cwd()`.
+ */
+function makeFilePathRelativeToCwd (file) {
+  return (file.startsWith(cwd)) ? '.' + file.substring(cwd.length) : file;
+}
+
+/**
+ * @param {String} outputDir the directory that contains output files
+ * @param {String[]} entryNames the entrypoint names from which output was produced
+ */
+async function zipOutputFiles (outputDir, entryNames) {
```

Comment:
It's really a bummer that all this logic needs to be added to zip files. Re-using a plugin like ZipPlugin allowed us to avoid maintaining this code. Is it really not possible to re-use other peoples code? Even spending an hour or so learning or patching ZipPlugin would be better than maintaining this for the rest of time =)

Reply:
There will be projects that have multiple lambdas but, also, by having support for `./src/lambas` as input, you can more easily copy scripts/configuration from project to project and you will have better consistency. I like having a convention that you put your lambda functions at `./src/lambas` so we can easily teach this and suggest a single build command.